### DISCUSSION TITLE
Stop Windows flashing carrier firmware

### DISCUSSION THREAD

**Discussion by jb455** [danielewood/sierra-wireless-modems#148] [Q&A]
Hi, I have an EM7455 on a Windows device and I'm trying to get 4G and GPS working.
I had it all ok with my test SIM card on generic FW but now I have the client-provided Vodafone SIM, and Windows keeps "helpfully" flashing the Vodafone FW which is old and I can't get the GPS to fix on it.
I've tried setting AT!IMPREF="GENERIC" but Windows flashes the Vodafone FW after at!reset or PC reboot.
Does anyone know if it's possible to either set a flag on the modem to lock firmware updates, or to stop Windows from getting in the way?

Thanks!

**Comment by adi8900** [comment]
Well here windows wants replacing firmware as well... I debranded from dell to sierra and installed
 SWI9X30C_02.33.03.00 fw as new 02.38.00.00 lacks some command support like at!lteca
You can hide this update which flashing another firmware with https://www.majorgeeks.com/files/details/wushowhide.html but u probably solved it already lmao
